### ISSUE TITLE
Debug view

### ISSUE THREAD

**Issue by saitonakamura** [pmndrs/react-three-flex#50]
What is the best way to debug why Boxes/children ended in places where they ended. I'm trying yo understand what gone wrong in my markup and having wireframe of flex container/highlight of the margins would be very helpful. But I don't understand how to do this properly (because of center anchor, for instance you can't set this on a flex). Is there any example I can look up?

**Comment by saitonakamura** [comment]
Ok, I think I got, I needed centerAnchor on a box (since it uses native three geometries). But it will still be nice to have this built-in

**Comment by ChrisCrossCrash** [comment]
I was going to make a new issue for exactly this! It would be very helpful if you could just put a `debug` prop in the root `Flex` element. Something like this:

```jsx
<Flex debug>
   ...
</Flex>
```

Then it would put wireframes around all the elements and flex containers. It could also show the anchors.

I've been trying to learn this library all day today, but it has been very difficult because there isn't an easy way to visualize the flex layout.

**Comment by proelf98** [comment]
Yes would love it. Would it be possible to set a material to the <Flex> Container?
